File: accidentes/FicheroAccidentes-Java-IvanRC/src/main/java/dto/accidente/ListaAccidentesDto.java
package dto.accidente;

import com.squareup.moshi.Json;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Root(name = "accidentes")
public class ListaAccidentesDto {
    @Json(name = "accidentes")
    @ElementList(name = "accidentes", inline = true)
    List<AccidenteDto> accidentesDto;
}

File: accidentes/FicheroAccidentes-Kotlin-IvanRC/src/main/kotlin/dto/accidente/ListaAccidentesDto.kt
package dto.accidente

import com.squareup.moshi.Json
import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

@Root(name = "accidentes")
data class ListaAccidentesDto(
    @Json(name = "accidentes")
    @field:ElementList(name = "accidentes", inline = true)
    @param:ElementList(name = "accidentes", inline = true)
    val accidentesDto: List<AccidenteDto>
)